README.md

```diff
@@ -287,11 +307,79 @@ Removes the file marker.
+| name         |                String               |   **Yes (resource name)**   |                                             | Name of the app to install.                                                                                                                                                                                                              |
+| package      | `:splunk` or `:universal_forwarder` |              No             | Package of the current install in run state | The installed Splunk package. If you did not specify the install\_dir, then you may specify the package (`:splunk` or `:universal_forwarder`) otherwise the resource will refer to the most recently evaluated splunk\_install resource  |
+| install\_dir |                String               |              No             | Path of the current install in run state    | The install directory of Splunk. If you did not specfiy the package, then you may specify the install\_dir otherwise the resource will refer to the most recently evaluated splunk\_install resource                                     |
+| source\_url  |                String               |              No             |                                             | **Not Implemented** Source url to use for installing a package or Git repo.                                                                                                                                                              |
+| version      |                String               | **Yes if app is versioned** |                                             | Version of the app to install. This is required if the app has a version in its app.conf, and will only install/upgrade if the version is different or the app is not already installed. If the app has no version, this does nothing.   |
+| configs      |                 Proc                |              No             |                                             | Proc containing `splunk_conf` resources. The resources will behave similarly to normal, except the config will be placed relative to the app directory rather than relative to the Splunk installation, and you can not change the scope. |
+| files        |                 Proc                |              No             |                                             | Proc containing miscellaneous resources for manipulating the app directory. This is ideally directories, templates, and files. The Proc will be given an absolute app path parameter for you to use in these resources.                  |
+| metadata     |                 Hash                |              No             | {}                                          | Configuration to apply to the metadata (local.meta in most cases, default.meta for custom apps). See splunk\_conf for more info, and also below for special handling of the `access` key.                                                |
```

Comment:
If it's an app we're creating on the fly, then we would edit default.meta (and /default configs), if it's a packaged app or app we're downloading from another source, then local is what we use: ( to quote @duckfez parodying Arthur C Clarke - "all these worlds are yours, except /default - attempt no editing there" )